### PR TITLE
Improve fact dl3

### PR DESCRIPTION
**Pull request by maxnoe**
* Use ErfaAstromInterpolator for faster coordinate transformations in fact_to_dl3
* Add az_prediction, zd_prediction columns (also for mc)
* Fix `parallelize_array_computation` for n_job=1
* Improve concatenating astropy time objects.

Verified everything is working by running the open crab sample analysis:

theta2_plot

### CODE REVIEW

**Comment by maxnoe** [comment]
the optimizations result in a nice factor of two improvement  for the application on the open crab data, with the main time now spend in applying the machine learning models.

**Review by LukasNickel — APPROVED** [review]

**Comment by LukasNickel** [comment]
There actually seems to be no astropy version for python 3.6.
Do we want to keep on supporting 3.6?
https://docs.astropy.org/en/stable/install.html

**Comment by maxnoe** [comment]
@LukasNickel, no we should just drop it. All major packages did that in their most recent versions following numpy:
https://numpy.org/neps/nep-0029-deprecation_policy.html

File: .travis.yml [hunk 1]
```diff
@@ -33,7 +32,7 @@ deploy:
   on:
     branch: master
     tags: true
-    condition: $TRAVIS_PYTHON_VERSION = '3.7'
+    condition: $TRAVIS_PYTHON_VERSION = '3.8'
```
Comment (LukasNickel) [inline .travis.yml:35]: Why do you need to up this as well?

Comment (maxnoe) [reply .travis.yml:35]: We don't. I just bumped everything by 1

**Review by LukasNickel — APPROVED** [review]

**Pull request merged**